src/sentry/types/issues.py

```diff
@@ -0,0 +1,34 @@
+from enum import Enum
+
+
+class GroupType(Enum):
+    ERROR = 1
+    PERFORMANCE_N_PLUS_ONE = 1000
+    PERFORMANCE_SLOW_SPAN = 1001
+
+
+class GroupCategory(Enum):
+    ERROR = 1
+    PERFORMANCE = 2
+
+
+GROUP_TYPE_TO_CATEGORY = {
+    GroupType.ERROR: GroupCategory.ERROR,
+    GroupType.PERFORMANCE_N_PLUS_ONE: GroupCategory.PERFORMANCE,
+    GroupType.PERFORMANCE_SLOW_SPAN: GroupCategory.PERFORMANCE,
+}
+
+
+def get_category_type_mapping():
+    category_type_mapping = {}
+
+    for type, category in GROUP_TYPE_TO_CATEGORY.items():
+        if category_type_mapping.get(category):
+            category_type_mapping[category].append(type)
+        else:
+            category_type_mapping[category] = [type]
```

Comment:
Nit: You can simplify this a bit with either
```
category_type_mapping = defaultdict(list)
for type, category in GROUP_TYPE_TO_CATEGORY.items():
    category_type_mapping[category].append(type)
```
or 
```
for type, category in GROUP_TYPE_TO_CATEGORY.items():
    category_type_mapping.setdefault(category, []).append(type)
```

Reply:
Let me try that, I was using a defaultdict yesterday and got very confused why it wasn't working. `setdefault` is new to me so I'll play w/ that too.